### ISSUE TITLE
VI half-field timing and initialization

### ISSUE THREAD

**Issue by cxd4** [project64/project64#157]
With a pixel-accurate plugin, interlaced video blitting works now with games running in 640x480 interlaced.  In some cases, there remains a particular problem with a game such as _Star Wars:  Rogue Squadron_.  If you have Counter Factor set to 1 then the problem is not really visible.  It is a much more visible problem with the default setting of CF=2, where interlaced DAC randomly fails and succeeds every other few frames.

The problem was that when I committed a fix for this (which zilmar adopted as the variable name `m_FieldSerration` within his class), I did not force the resetting of the serration status to 0 when the next interrupt is initialized.

Basically the code is correct for the most part:
```c
void CMipsMemoryVM::UpdateFieldSerration (unsigned int interlaced)
{
	m_FieldSerration ^= 1;
	m_FieldSerration &= interlaced;
}
```
However, this could cause m_FieldSerration to start at 1 instead of 0, when it maybe should be 0 instead of 1, depending on the CPU timing and at what stage the VI interval is at when the current video interrupt began drawing the first scanline of the frame.

I will look at it later to see how to equate this fix to a pj64 version.

**Comment by cxd4** [comment]
Never mind.  I thought I was getting to clear the LSB per some stage of groups of vertical interrupts, but really that was getting done on ROM initialization which was enough.

The implementation is about optimal from non-cycle-accurate standpoint.  Any issues past that need to be filed as pull requests for counter factor or VI Refresh Rate.
